src/DynamoCore/Graph/Nodes/NodeInputData.cs

```diff
@@ -22,7 +22,9 @@ public enum NodeInputTypes
         [EnumMember(Value = "date")]
         dateInput,
         [EnumMember(Value = "selection")]
-        selectionInput
+        selectionInput,
+        [EnumMember(Value = "itemSelection")]
```

Comment:
what is  the difference between itemSelection and selection? I think selection also just returns a string...

Reply:
I think the idea here was to have a way of distinguish between selecting something in the host environment (i.e. `Select Model Elements`) and selecting an item from a list (i.e. a dropdown selection).

@saintentropy can probably elaborate on this.

Reply:
@saintentropy and I just talked about this for a bit. The real issue is that the existing "selection" \ `selectionInput` type is ambiguously named, and in a Player UI context it could be interpreted as either a button (like a "Select Elements" button in the Revit context) or as a dropdown list.

@Dewb suggested that we just deprecate the existing "selection" member, and introduce two new members, one for buttons and one for dropdowns. I think "hostElementSelection" or just "hostSelection" for the first, and "listSelection" or "dropdownSelection" for the second.